main.go

```diff
@@ -79,6 +85,14 @@ func ParseArgs() (string, []string) {
 		Help("help")
 	}
 
+	if os.Args[1] == "version" || os.Args[1] == "-v" || os.Args[1] == "--ver" {
```

Comment:
is `--ver` necessary? It doesn't seem very common.

Reply:
Could go without it. A third opinion to rule it?

Reply:
Only `version` or `--version` should be necessary.

Reply:
Maybe just remove it.